src/App.tsx

```diff
@@ -1,109 +1,14 @@
 import React from 'react';
 
 import './App.scss';
```

Comment:
Extract styles for every component and move to proper files. Every component has its own styles.